packages/shared/warningWithoutStack.js

```diff
@@ -25,16 +25,18 @@ if (__DEV__) {
     if (condition) {
       return;
     }
+
+    let argIndex = 0;
+    const message =
+      'Warning: ' + format.replace(/%s/g, () => `${args[argIndex++]}`);
+
     if (typeof console !== 'undefined') {
-      console.error('Warning: ' + format, ...args);
+      console.error(message);
```

Comment:
The part about passing `...args` *is* intentional. It lets other tools read the stack separately.

I'm only suggesting to map over them and coerce them to strings.